### DISCUSSION TITLE
Implementing Neumann BC but on a function of u(x,y)

### DISCUSSION THREAD

**Discussion by SebastianFerra** [lululxvi/deepxde#1919] [Q&A]
Hi! I'm trying to implement a boundary conditions that looks like  P\cdot n = 0, where P is a function of u which is the function we are solving for, I've tried with NeumannBC and OperatorBC but it isn't all that clear how to implement it, i would appreciate your help!

**Comment by lululxvi** [comment]
This is not classical BC. You need to implement it yourself. You can check the code of NeumannBC on how to compute n.
